org.nickvision.cavalier.json

```
@@ -0,0 +1,92 @@
+{
+    "app-id" : "org.nickvision.cavalier",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "command" : "/app/bin/org.nickvision.cavalier",
```

it's never a path...

```suggestion
    "command" : "org.nickvision.cavalier",
```